### PR TITLE
Don't bundle unnecessary plugins in `@babel/standalone`

### PR DESCRIPTION
**Pull request by nicolo-ribaudo**
| Q                        | A 
| ------------------------ | ---
| Fixed Issues?            | 
| Patch: Bug Fix?          |
| Major: Breaking Change?  |
| Minor: New Feature?      |
| Tests Added + Pass?      | Yes
| Documentation PR Link    | 
| Any Dependency Changes?  |
| License                  | MIT

`@babel/standalone` included different syntax plugins that were not needed, because the bundled `@babel/parser` version has them enabled by default. Also, we can avoid bundling `@babel/helper-plugin-utils`.

<a href="https://gitpod.io/#https://github.com/babel/babel/pull/15023"></a>

### CODE REVIEW

File: scripts/rollup-plugin-standalone-internals.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  fs.readFileSync(new URL("./scripts/pluginConfig.json", standaloneURL), "utf8")
+);
+
+const pluginUtilsID = "@babel/helper-plugin-utils";
+
+const generatedPluginsPath = inStandalone("./src/generated/plugins.ts");
+const makeNoopPluginPath = inStandalone("./src/make-noop-plugin.ts");
+const pluginUtilsShimPath = inStandalone("./src/plugin-utils-shim.ts");
+
+const toCamel = str => str.replace(/-[a-z]/g, c => c[1].toUpperCase());
```
Comment (JLHwung) [inline scripts/rollup-plugin-standalone-internals.js:16]:
```suggestion
const toCamel = str => str.replaceAll(/-[a-z]/g, c => c[1].toUpperCase());
```
nit. And should this file be an `.mjs` file?

**Review by JLHwung — APPROVED** [review]

File: scripts/rollup-plugin-standalone-internals.js (hunk 1 above)
Comment (nicolo-ribaudo) [reply scripts/rollup-plugin-standalone-internals.js:16]: There is a `package.json` with `"type": "module"`

**Comment by nicolo-ribaudo** [comment]
@liuxingbaoyu Could you help me with the windows failure? :pray:

**Comment by liuxingbaoyu** [comment]
Sure.

**Pull request merged**